fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -832,7 +833,7 @@ public class Config extends ConfigBase {
      * Parallel load fragment instance num in single host
      */
     @ConfField(mutable = true)
-    public static int load_parallel_instance_num = 1;
+    public static int load_parallel_instance_num = BackendCoreStat.getAvgNumOfHardwareCoresOfBe();
```

Comment:
I'm afraid that `BackendCoreStat.getAvgNumOfHardwareCoresOfBe()` can return valid value when `load_parallel_instance_num` variable initialized.

Reply:
oh, nice catch. 
Shall we use a constant? say, 4, or 8?
would like to increase the default parallelism. Currently, the default load sometimes slow :)